tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -73,8 +74,39 @@ spec:
   ovnContainerImage: quay.io/podified-antelope-centos9/openstack-ovn-controller:current-podified
   ovsContainerImage: quay.io/podified-antelope-centos9/openstack-ovn-base:current-podified
 status:
-  desiredNumberScheduled: 1
-  numberReady: 1
+  conditions:
+  - message: Setup complete
```

Comment:
that's fine though I wonder if we have to check all of the conditions; I think Ready would be enough (I don't know if kuttl will allow to check just the important one though, and there's no significant harm in checking more, so feel free to ignore).

Reply:
Since this is a list, kuttl needs taht all conditions are there, otherwise it compares the list length and fails. We could however remove fields, like the `message` or `reason` to make it less verbose